fuzz/writeout.c

```diff
@@ -177,7 +183,12 @@ void write_out_track(ASS_Track *track, const char *outpath)
         printf("Parsed File will be written to:  %s\n", outpath);
     } else {
         char filename[] = "/tmp/parsedSubs_XXXXXX";
+#ifdef _WIN32
+        _mktemp(filename);
+        int fd = _open(filename, O_CREAT | _O_EXCL | _O_WRONLY, _S_IREAD | _S_IWRITE);
+#else
         int fd = mkstemp(filename);
```

Comment:
Unrelated note, but for fuzzing on Linux it is better to use `memfd_create` in such cases. See for example https://github.com/mpv-player/mpv/blob/baf528069a584c04686262ef6f76e2c9232adba3/fuzzers/fuzzer_load.c#L45-L58

Reply:
This code is not even compiled in for continuous fuzzing builds and `memfd_create` defeats the purpose of the writeout functionality